Command/LoadYamlFixturesCommand.php

```diff
@@ -66,8 +65,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if ($input->isInteractive() && !$input->getOption('append')) {
-            /* @var DialogHelper $dialog */
-            $dialog = $this->getHelperSet()->get('dialog');
+            $helperName = class_exists('Symfony\Component\Console\Helper\QuestionHelper') ? 'question' : 'dialog';
```

Comment:
@mhlavac is it available since symfony 2.3 (since it is oldest currently supported version)?